WireAPI/Sources/WireAPI/APIs/ConversationsAPI/ConversationsAPIV5.swift

```diff
@@ -55,7 +59,7 @@ class ConversationsAPIV5: ConversationsAPIV4 {
             throw ConversationsAPIError.userAndDomainShouldNotBeEmpty
         }
 
-        let components = URLComponents(string: "\(pathPrefix)\(basePath)/one2one/\(domain)/\(userID)")
+        let components = URLComponents(string: "\(oneToOneConversationsPath)/\(domain)/\(userID)")
```

Comment:
I wanted to fix this path, but then I had to make all the other changes as well (extending APIVersion etc.).